integration-test-server-stats.sh

```
@@ -0,0 +1,808 @@
+    
+    # Start background processes that consume some CPU
+    for i in {1..2}; do
+        (while true; do echo "load test" >/dev/null; done) &
+        load_pids+=($!)
+    done
+    
+    # Give the load processes a moment to start
+    sleep 1
+    
+    # Test script execution under load
+    local start_time=$(date +%s.%N)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)